src/pages/home/report/ReportActionItem.js

```diff
@@ -415,7 +415,7 @@ function ReportActionItem(props) {
     }
 
     const hasErrors = !_.isEmpty(props.action.errors);
-    const whisperedTo = lodashGet(props.action, 'whisperedTo', []);
+    const whisperedTo = props.action.whisperedTo || [];
```

Comment:
Why this change?

Reply:
Lodash get is unnecessary here and this is faster and just as safe. Tim taught me that.